Makefile

```diff
@@ -46,3 +46,7 @@ check-latest-imagestream:
 
 check-betka:
 	cd tests && ./check_betka.sh
+
+push-to-containers:
+	@echo "THIS COULD BE DANGEROUS, WILL PUSH TO ALL SCLORG CONTAINER REPOSITORIES"
+	./push_as_submodule.sh
```

Comment:
Do we really want to name it push_as_submodule?
I would propose something like `push_sclorg_repos`.

Or is this script closely related to only common? I guess we would like to use it also e.g. Update GitHub actions ;)

Reply:
Yes. I can not imagine any other situation (than updating submodules), that will require pushing the same patch to more container repositories. The repositories are too different and the patch would not be applicable in the vast majority of cases.